### ISSUE TITLE
isiXhosa

### ISSUE THREAD

**Issue by zakwaxhosa** [common-voice/common-voice#3030]
Use this template to request a new localizable language that is currently not available [on Pontoon](https://pontoon.mozilla.org/projects/common-voice/).

Note: This issue only applies to the web interface language - in order to activate language contributions on Common Voice you will also need to ensure 5,000+ sentences are available to be read in that language. Please refer to the [full Language documentation](https://github.com/mozilla/common-voice/blob/master/README.md) for more details.

**Language name**
What language would you like to add?
isiXhosa
**Language code**
Please provide the ISO-639-1 code
XH
**Language size**
Number of active speakers of this language in the world
10 MILLION 
**Plural forms**
How would you translate the following in this language?

0 rocks  AMATYE  
1 rock  ilitye  elinye 
2 rocks amatye amabini
3 rocks amatye amathathu 
4 rocks amatye amane
5 rocks amatye amahlanu
10 rocks amatye alishumi
20 rocks amatye anagamashumi amabini
100 rocks amatye alikhulu
1000 rocks amatye aliwaka
I see 0 rocks on the ground andiboni matye phantsi 
I see 1 rock on the ground ndibona ilitye elinye phantsi 
I see 10 rocks on the ground ndibona amatye alishumi phantsi 
I see rocks on the ground ndibona amatye phantsi 

**Pontoon manager**
Please link to the Pontoon account of the person who should be listed as the community manager of this language

**Comment by ftyers** [comment]
Dear @zakwaxhosa, [Xhosa](https://pontoon.mozilla.org/xh/) is already in Pontoon, so you don't need to enable it there. You will need to apply for the project to be enabled through the Pontoon app. @peiying2 could you explain how?

Also note that Xhosa is already enabled in [sentence collector](https://commonvoice.mozilla.org/sentence-collector/) so you won't need to ask for it to be enabled, you can just start submitting and reviewing sentences there. You will need to add 5000 before it can be enabled in the main site. There are further details [here](https://github.com/common-voice/common-voice/blob/main/docs/LANGUAGE.md). Please feel free to contact us if you have any questions.

**Comment by peiying2** [comment]
> Dear @zakwaxhosa, [Xhosa](https://pontoon.mozilla.org/xh/) is already in Pontoon, so you don't need to enable it there. You will need to apply for the project to be enabled through the Pontoon app. @peiying2 could you explain how?

Yes, the language  already exists so I consider  this an official request. @zakwaxhosa could you tell me if you will be leading  the effort to localize the project. Do you have someone to work with,  to review your work? Do you mind sharing a little bit of your background so I can decide what role to  assign you? The community has been active for years. Are you ok to be the locale manager? Or are you only interested in being a manager? 

Please create an account in Pontoon so I can set you up. Then I will share some basic documentations with you. Thank  you!!

**Comment by phirework** [comment]
Hi @zakwaxhosa, are there any updates to @peiying2's request? Otherwise I will close this issue.

**Comment by zakwaxhosa** [comment]
Hi

Thank you for the feedback .

I wanted to contribute with sentence review and submissions by reading our
Constitution in Xhosa but I just can't get to Xhosa on GitHub .

@ peiying ,yes pls make this an official request .

I'm a native Xhosa speaker ,working in  the financial sector .

Regards

**Comment by peiying2** [comment]
@zakwaxhosa could you send me the URL of your Pontoon Profile so I can add you as a translator in Xhosa in Pontoon. Thank you.

**Comment by Heyhillary** [comment]
Hey @zakwaxhosa 

Thanks for the request to include isiXhosa. I'm messaging to follow up to see if you are still interested in adding isiXhosa.?
If you are pleased, share with us the URL link for the Pontoon Profile. 

If you have any questions please feel free to ask.

Kind regards,

Hillary

**Comment by moz-bozden** [comment]
The language has already been launched.
Closing
